coordinator/src/node.rs

```diff
@@ -24,19 +24,32 @@ use ln_dlc_node::node::SubChannelManager;
 use ln_dlc_node::PeerManager;
 use rust_decimal::prelude::ToPrimitive;
 use rust_decimal::Decimal;
+use std::collections::HashMap;
 use std::sync::Arc;
+use std::sync::Mutex;
 use trade::cfd;
 use trade::cfd::calculate_long_liquidation_price;
 use trade::cfd::calculate_margin;
 use trade::cfd::calculate_short_liquidation_price;
 use trade::cfd::BTCUSD_MAX_PRICE;
+use trade::ContractSymbol;
 use trade::Direction;
 
 /// The leverage used by the coordinator for all trades.
 const COORDINATOR_LEVERAGE: f64 = 1.0;
 
 pub struct Node {
     pub inner: Arc<ln_dlc_node::node::Node>,
+    pub positions: Mutex<HashMap<String, Position>>,
+}
+
+pub struct Position {
+    pub contract_symbol: ContractSymbol,
+    pub leverage: f64,
+    pub quantity: f64,
+    pub direction: Direction,
+    pub trader: PublicKey,
+    pub average_entry_price: f64,
```

Comment:
🙃 Could just store the `average_entry_price` since that all we care about and this is just a band-aid fix.